### PR TITLE
Add company-facing resume portal

### PR DESCRIPTION
**Pull request by kevinschoonover**
# Problems
Add portal for companies to view student-uploaded resumes

closes: #11

### CODE REVIEW

File: apps/resume-portal/.gitignore [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```
Comment (ClayMav) [inline apps/resume-portal/.gitignore]: tsk tsk, this is wrong, should be
```suggestion
node_modules/
```

File: apps/resume-portal/.gitignore [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+/.pnp
```
Comment (ClayMav) [inline apps/resume-portal/.gitignore]:
```suggestion
.pnp/
```

File: apps/resume-portal/.gitignore [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+.pnp.js
+
+# testing
+/coverage
```
Comment (ClayMav) [inline apps/resume-portal/.gitignore]:
```suggestion
coverage/
```

File: apps/resume-portal/.gitignore [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+# production
+/build
```
Comment (ClayMav) [inline apps/resume-portal/.gitignore]:
```suggestion
build/
```

File: apps/resume-portal/public/manifest.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```
Comment (ClayMav) [inline apps/resume-portal/public/manifest.json]: You are better than this

File: apps/resume-portal/src/App.css [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
```
Comment (ClayMav) [inline apps/resume-portal/src/App.css]: Most of this file should be gone

**Comment by kevinschoonover** [comment]
Closing in favor of new branch as this is too out of date

**Pull request closed without merging**